NewHorizons/Builder/Volumes/VolumeBuilder.cs

```diff
@@ -0,0 +1,31 @@
+using NewHorizons.Components;
+using NewHorizons.External.Modules;
+using UnityEngine;
+
+namespace NewHorizons.Builder.Volumes
+{
+    public static class VolumeBuilder
+    {
+        public static TVolume Make<TVolume>(GameObject planetGO, Sector sector, VolumesModule.VolumeInfo info) where TVolume : MonoBehaviour
```

Comment:
Can't TVolume : BaseVolume ?

Reply:
Yes, but I need to create mobius volumes too.

Reply:
got it